app/questionnaire/placeholder_transforms.py

```diff
@@ -125,6 +125,36 @@ def parse_date(date):
                 date, PlaceholderTransforms.input_date_format_month_year_only
             ).replace(tzinfo=tzutc())
 
+    @staticmethod
+    def add(lhs, rhs):
+        if isinstance(lhs, str):
```

Comment:
Why only the cast on the first value? surely they both could be strings, or even something else

Reply:
I changed it so it checks both now, thanks!